azkaban-common/src/test/java/azkaban/executor/mail/DefaultMailCreatorTest.java

```diff
@@ -122,8 +121,7 @@ public void createSuccessEmail() throws Exception {
         this.executableFlow, this.message, this.azkabanName, this.scheme, this.clientHostname,
         this.clientPortNumber));
     assertEquals("Flow 'mail-creator-test' has succeeded on unit-tests", this.message.getSubject());
-    assertEquals(read("successEmail.html").replaceAll("\\s+", ""),
-        this.message.getBody().replaceAll("\\s+", ""));
+    assertThat(read("successEmail.html")).isEqualToIgnoringWhitespace(this.message.getBody());
```

Comment:
Thanks. This looks much better.

Is http://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/AbstractCharSequenceAssert.html#isEqualToNormalizingWhitespace(java.lang.CharSequence) more appropriate?